alpha_importer.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import print_function
+from redis import Redis
+import config
+import requests
+from requests.exceptions import HTTPError
+import ujson
+import time
+import sys
+import logging
+from logging.handlers import RotatingFileHandler
+import os
+
+redis_connection = Redis(host = config.REDIS_HOST)
+QUEUE_KEY = 'alphaimporter:queue'
+SET_KEY_PREFIX = 'alphaimporter:set'
+ALPHA_URL = 'https://alpha.listenbrainz.org/'
+BETA_URL = 'https://listenbrainz.org'
```

Comment:
I know I said that we didn't need to have these in the config file, but now that I am testing, it is clear they should be there. Can you please add these to the config.py file so we can change them easily for testing?